### DISCUSSION TITLE
Errorbars

### DISCUSSION THREAD

**Discussion by asandrock** [kookma/ogpf#28] [Ideas]
How do I plot errorbars with ogpf? In gnuplot, this is possible via e.g. "plot 'a.file' u 1:2:3 with errorbars", "plot 'a.file' u 1:2:3 with xerrorbars", "plot 'a.file' u 1:2:3 with yerrorbars" or "plot 'a.file' u 1:2:3:4 with xyerrorbars".

**Comment by kookma** [comment]
I will add this to enhancement and will add examples!
